projects/schematics/src/migrations/4_0/methods-and-properties-deprecations/data/popover-directive.migration.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  HANDLE_OPEN,
+  POPOVER_DIRECTIVE,
+  SPARTACUS_STOREFRONTLIB,
+  TODO_SPARTACUS,
+  TOGGLE,
+} from '../../../../shared/constants';
+import { MethodPropertyDeprecation } from '../../../../shared/utils/file-utils';
+
```

Comment:
Same here. We can add a comment with path to file to be consistent with other files.